resources/assets/js/fieldtype.js

```diff
@@ -137,6 +149,7 @@ Vue.component('video_embed-fieldtype', {
                     that.data.thumbnail_large = data.items[0] ? data.items[0].snippet.thumbnails.high.url : '';
                     that.data.thumbnail_medium = data.items[0] ? data.items[0].snippet.thumbnails.medium.url : '';
                     that.data.thumbnail_small = data.items[0] ? data.items[0].snippet.thumbnails.default.url : '';
+                    that.data.duration = data.items[0] ? data.items[0].contentDetails.duration : '';
```

Comment:
The time needs to be converted to seconds before it is assigned to `that.data.duration`